### DISCUSSION TITLE
Breeze Evolution as parent theme

### DISCUSSION THREAD

**Discussion by 2EyeTec** [breezefront/community#133] [1. Help] [answered]
I'm trying to inherit Breeze Evolution, but I when I compile the static content, I receive a lot of errors "Unable to resolve the source file". I'm able to achieve what I want by inheriting the Breeze Blank theme and copying vendor/swissup/theme-frontend-breeze-evolution to the folder of my theme. But actually I want only the files in this place which I changed to my needs.
I'm wondering if it is not possible to inherit Breeze Evolution or not in the way inheriting Breeze Blank or am I doing something wrong?

**Comment by vovayatsyuk** [comment, marked as answer]
BreezeEvolution/Blank are just regular magento themes. You can create as many child themes as you want.

Please do not copy everything into your child's theme. By doing this you will not be able to upgrade your store easily.

How to create a child theme: https://breezefront.com/docs/child-theme
How to change theme styles: https://breezefront.com/docs/custom-styles
